Makefile

```diff
@@ -21,7 +21,6 @@ help: ## Show this help
 ---------------: ## ---------------
 
 up: ## Start all containers (in background) for development
-	sudo sysctl -w vm.max_map_count=262144
```

Comment:
I think we have  to leave it or check os before run
https://stackoverflow.com/questions/714100/os-detecting-makefile